rebasehelper/spec_content.py

```diff
@@ -104,6 +104,25 @@ def get_comment_span(cls, line: str, section: str) -> Tuple[int, int]:
         comment = re.search(r" #.*" if inline_comment_allowed else r"^\s*#.*", line)
         return comment.span() if comment else (len(line), len(line))
 
+    def nth_section(self, name: str, n: int) -> Optional[List[str]]:
+        """Gets content of n-th section of the given name.
+
+        Args:
+            name: Name of the section.
+            n: Number of the section with name to return.
+
+        Returns:
+            The found section or None if n-th section with name doesn't exist.
+
+        """
+        index = 0
+        for header, section in self.sections:
+            if header.lower() == name.lower():
+                if index == n:
+                    return section
+                index += 1
+        return None
+
```

Comment:
This is not what I had in mind. SpecContent should behave like an iterable and the index should be absolute index in that iterable. not relative to section name. That's confusing and unnecessarily complicated.